### PR TITLE
feat(nav): added necessery navigational components

### LINKED ISSUE #3: Create skeleton of application navigation (app routes)

### PR DESCRIPTION
**Pull request by lxndr**
Closes #3

There are two issues with the drawer component:
1. It lags a little when being open or closed. Looks fixed in `react-navigation-drawer@2.0.0-alpha.3`.
2. When the map is visible, right swipe moves the map instead of the drawer. There is [an example](https://github.com/react-navigation/drawer/blob/ed8dcb4335f551cfdb7f6d90ba876e300740f33b/example/src/GestureInteraction.js#L45) how to fix it the right way but it does not work.

### CODE REVIEW

File: client/src/screens/consts.ts [hunk 1]
```diff
@@ -1,2 +1,9 @@
-export const HOME = 'Home';
```
[the remark this answers is not in the document]
Comment (kuzn-ilya) [reply client/src/screens/consts.ts]: @lxndr , I agree with @eugrdn , could please make necessary changes.

[the remark this answers is not in the document]
Comment (lxndr) [reply client/src/screens/consts.ts]: Ok, is `Routes` too short and confusing?

[the remark this answers is not in the document]
Comment (kuzn-ilya) [reply client/src/screens/consts.ts]: I can live with it, it's ok for me

**Comment by lxndr** [comment]
So, let's merge it? Even though I'm not entirely happy with how the drawer works, we've got the basis to start implementing screens on.

**Pull request merged**